Dockerfile

```diff
@@ -44,10 +50,10 @@ ARG VITE_SELFHOSTED="true"
 RUN pnpm install --frozen-lockfile
 
 # IDK why but it gets mad if you don't do this
-RUN git config --global --add safe.directory /app
+# RUN git config --global --add safe.directory /app
 
 # Build the static site
-RUN pnpm run build
+# RUN pnpm run build
```

Comment:
it is no longer building the website so that's why you're getting the route not found

Reply:
I tried to put that in the docker_entrypoint

Reply:
made some updates that got it working for me with the submodules

Reply:
you'll need to run `git submodule update`, we had to make a small change to mutiny-web

Reply:
works ™